src/js/helpers/elementCreatorHelper.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * A series of functions to assist in creating DOM elements
+ *
+ * Add more as needed & make sure to list them here :
+ *
+ * - createTitle
+ * - createContainer
+ */
+
+
+/**
+ * Create a h1 title
+ * @param {Object} args An object containing the options for creating the title.
+ * The accepted keys are :
+ *  @key {String} titleText The text of the title
+ *  @key {Array} classes The tailwind css classes to be applied to the title elements
+ *
+ * @return {DOM element} h1 The title
+ */
+export function createTitle(args) {
+
+  const {
+    titleText = 'Please provide a value for the titleText key',
+    titleType = 'h1' ,
+    classes = []
+  } = args;
+
+  const h1 = document.createElement(titleType);
+  h1.innerText = titleText;
+  h1.classList.add(...classes);
+
+  return h1;
```

Comment:
Rename this variable - it's a but confusing to call it h1, as really it could be any type of heading
```suggestion
  const heading = document.createElement(titleType);
  heading.innerText = titleText;
  heading.classList.add(...classes);

  return heading;
```

Reply:
Fine by me